.setup/CONFIGURE_SUBMITTY.sh

```diff
@@ -51,8 +48,8 @@ SUBMITTY_INSTALL_DIR=/usr/local/submitty
 SUBMITTY_DATA_DIR=/var/local/submitty
 
 # Log locations
-TAGRADING_LOG_PATH=$SUBMITTY_DATA_DIR/tagrading_logs/
-AUTOGRADING_LOG_PATH=$SUBMITTY_DATA_DIR/autograding_logs/
+TAGRADING_LOG_PATH=$SUBMITTY_DATA_DIR/logs/site/
```

Comment:
Do we need the "site" folder?
I suggest:
/var/local/submitty/logs/autograding
/var/local/submitty/logs/access
/var/local/submitty/logs/error